src/lib/stores/api.js

```diff
@@ -300,23 +300,17 @@ export default {
     let userCredential;
     try {
       userCredential = await signInWithEmailAndPassword(auth, email, password);
+      window.location.href = "/studies";
     } catch (err) {
       console.error("there was an error", err);
       localStorage.setItem("signInErr", err);
       return;
     }
-    if (userCredential.user.emailVerified) {
-      initializeUserDocument(userCredential.user.uid);
-      listenForUserChanges(userCredential.user);
-      listenForUserStudiesChanges(userCredential.user);
-
-      // Let the Rally SDK content script know the site is intialized.
-      console.debug("initialized, dispatching rally-sdk.web-check");
-      window.dispatchEvent(new CustomEvent("rally-sdk.web-check", {}));
-    } else {
+    if (!userCredential.user.emailVerified) {
       console.warn("Email account not verified, sending verification email");
       localStorage.setItem("signInErr", "Email account not verified");
       await sendEmailVerification(userCredential.user);
+      await this.signOutUser();
```

Comment:
This is fairly minor, but the way this works is that users can log in, but nothing works until they verify their email address. We *could* allow them to stay logged in and have their session just start working, but we're not at all set up to support that right now, so let's just log them out if they aren't verified.

Reply:
The only thing here is an option for users to resend confirmation email. That won't work if we sign out I think.